tests/robot-tests/tests/admin_and_public_2/bau/publish_methodology_publication_update.robot

```diff
@@ -133,6 +133,7 @@ Update methodology details
     user enters text into element    id:updateMethodologyForm-title    ${PUBLICATION_NAME}-methodology update
     user waits until button is clickable    Update methodology
     user clicks button    Update methodology
+    user reloads page
```

Comment:
@saicharan2789 can we use `user waits until page finishes loading` here as well?

Reply:
done